release_notes.md

```diff
@@ -1,3 +1,93 @@
+keyword arguments `is_indep_var` and `is_design_var` to help users filter inputs.
+POEM 082 adds a similar method, `Problem.get_indep_vars` which will allow the user to find all variables in the problem
+that are provided by IndepVarComps (or systems that mimic them). POEM 076 implements directional check_totals to provide
+a faster way of derivative checking in very large models.
+
+OptionsDictionary now supports a `set` method to allow users to set multiple options in a single line.
+
+
+## New Deprecations
+
+- None
+
+## Backwards Incompatible API Changes
+
+- Removed deprecation that allowed use of `value` in place of `val` [#2835](https://github.com/OpenMDAO/OpenMDAO/pull/2835)
+- Removed deprecated simple_warning() function [#2837](https://github.com/OpenMDAO/OpenMDAO/pull/2837)
+- Removed deprecated DemuxComp [#2841](https://github.com/OpenMDAO/OpenMDAO/pull/2841)
+- Removed deprecated 'factorial' support in ExecComp [#2842](https://github.com/OpenMDAO/OpenMDAO/pull/2842)
+- Changed step_calc 'rel' to act the same as 'rel_avg' and removed deprecation [#2846](https://github.com/OpenMDAO/OpenMDAO/pull/2846)
+- Removed deprecated OPENMDAO_REQUIRE_MPI check [#2848](https://github.com/OpenMDAO/OpenMDAO/pull/2848)
+- Removed deprecated assert_rel_error() function [#2849](https://github.com/OpenMDAO/OpenMDAO/pull/2849)
+- Removed deprecated get_conversion() utility function [#2850](https://github.com/OpenMDAO/OpenMDAO/pull/2850)
+- Removed deprecated CaseReader attributes [#2851](https://github.com/OpenMDAO/OpenMDAO/pull/2851)
+- Removed deprecated `indep_var` tag [#2854](https://github.com/OpenMDAO/OpenMDAO/pull/2854)
+- Removed deprecated names for Akima1D and trilinear interpolants [#2855](https://github.com/OpenMDAO/OpenMDAO/pull/2855)
+- Removed deprecated ability to specify src_indices as an argument to add_input() [#2864](https://github.com/OpenMDAO/OpenMDAO/pull/2864)
+- Removed ability to use a Component as Problem.model [#2869](https://github.com/OpenMDAO/OpenMDAO/pull/2869)
+
+## Backwards Incompatible Non-API Changes
+
+- None
+
+## New Features
+
+- Changed DOEDriver to record unscaled derivatives [#2811](https://github.com/OpenMDAO/OpenMDAO/pull/2811)
+- POEM 78: Add is_indep_var and is_design_var filters to list_inputs/list_outputs [#2813](https://github.com/OpenMDAO/OpenMDAO/pull/2813)
+- During CS derivatives check, display Jcs instead of Jfd to let the user know the method used. [#2812](https://github.com/OpenMDAO/OpenMDAO/pull/2812)
+- Added ability to show/hide the promotion tree for each promoted input/output in the connection viewer [#2826](https://github.com/OpenMDAO/OpenMDAO/pull/2826)
+- Added capability to `list_problem_vars` to return problem var data and suppress `out_stream` [#2834](https://github.com/OpenMDAO/OpenMDAO/pull/2834)
+- Refactor for complex step around Newton. [#2839](https://github.com/OpenMDAO/OpenMDAO/pull/2839)
```

Comment:
"Refactor for complex step around Newton" --> Refactored how complex-step was performed around a Newton solver to address an accuracy problem.